website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+- Why is the dashboard filter saying that all User’s `Zones` are Zone 2?
+
+This familiarity with what types of data quality issues are the most important to the business means that an Analyst can often identify in advance which automated tests should be added to a data model.
+
+## Loop 3: Reacting to mismatched documentation
+
+There’s nothing worse than coming back to view a curated dataset months later (or maybe after someone on the team has offboarded) and learning that there is nothing written to explain why certain logic exists. Or worse, the documentation exists but it no longer matches what the model actually does. Which brings us to our third and final point:
+
+<h3 align="center">Analysts understand the pain of a poorly documented dataset.</h3>
+
+It’s much easier to keep to a naming guide when the writer has a deep understanding of the data and how it’s referenced by the business. Analysts are often asked repeat questions by stakeholders, which is extra incentive to create great documentation from the beginning.
+
+> **A Data Engineer will document a dataset based on what they needed to know to build it. An Analyst with Analytics Engineering skills will document a dataset based on *how to use it going forward.***
+
+If we want to know how certain logic was built technically, then we can reference the SQL code in dbt docs. If we want to know *why* a certain logic was built into that specific model, then that’s where we’d turn to the documentation.
+
+- Example of not-so-helpful documentation ([dbt docs can](https://docs.getdbt.com/docs/building-a-dbt-project/documentation) build this dynamically):
+    - `Case when Zone = 1 and Level like 'A%' then 'True' else 'False' end as GroupB`
+- Example of better, more descriptive documentation (add to your dbt markdown file or column descriptions):
+    - Group B is defined as Users in Zone 1 with a Level beginning with the letter 'A'. These users are accessing our new add-on product that began in Beta in August 2022. It's recommended to filter them out of the main Active Users metric.
+
+In the second example, the documentation is written with the stakeholder and business in mind, and Analysts are experts at this translation of the technical to the non-technical.
+
+## Are you convinced?
+
+Teach your Analyst to be an Analytics Engineer, so they can do their own model building for curated datasets. You’ll start to see their process evolve into multiple smaller iterations, as they add fields and test them as they build. No loops, no hand-offs, and no Telephone Game. Your new workflow will look a little like this:
+
+![Comparing workflows with and without engineers](/img/blog/2022-09-28-analyst-to-ae/comparison_workflow.png)
+
+Instead of trying to define all the various groups of `Active Users` at onc, an Analytics Engineer can be validating the `Group A` rows, while adding `Group B` to their local environment, while still working with the stakeholder on the definition of `Group C`.
```

Comment:
```suggestion
Instead of trying to define all the various groups of `Active Users` at once, an Analytics Engineer can be validating the `Group A` rows, while adding `Group B` to their local environment, while still working with the stakeholder on the definition of `Group C`.
```